controllers/suggestions/suggestions_review_base_controller.py

```diff
@@ -1,11 +1,17 @@
-from consts.account_permissions import AccountPermissions
-
 from controllers.base_controller import LoggedInHandler
 
+
 class SuggestionsReviewBaseController(LoggedInHandler):
     """
     Base controller for reviewing suggestions.
     """
+
+    REQUIRED_PERMISSIONS = []
+
     def __init__(self, *args, **kw):
         super(SuggestionsReviewBaseController, self).__init__(*args, **kw)
-        self._require_permission(AccountPermissions.MUTATE_DATA)
+        self.verify_permissions()
+
+    def verify_permissions(self):
+        for permission in self.REQUIRED_PERMISSIONS:
```

Comment:
I can imagine cases in the future where it is, so I made it flexible.

On Fri, May 13, 2016 at 9:12 AM Eugene Fang <notifications@github.com>
wrote:

> In controllers/suggestions/suggestions_review_base_controller.py
> <https://github.com/the-blue-alliance/the-blue-alliance/pull/1529#discussion_r63209829>
> :
>
> >      def __init__(self, *args, **kw):
> >          super(SuggestionsReviewBaseController, self).__init__(*args, **kw)
> > -        self._require_permission(AccountPermissions.MUTATE_DATA)
> > +        self.verify_permissions()
> > +
> > +    def verify_permissions(self):
> > +        for permission in self.REQUIRED_PERMISSIONS:
>
> Is REQUIRED_PERMISSIONS ever more than just one value?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/the-blue-alliance/the-blue-alliance/pull/1529/files/3d6aa6044cd6a21971f0a90d608dd82e775ac28f#r63209829>
>